content/docs/command-reference/import.md

```diff
@@ -276,3 +278,133 @@ outs:
 
 The `url` and `rev_lock` subfields under `repo` are used to save the origin and
 [version](https://git-scm.com/docs/revisions) of the dependency, respectively.
+
+## Example: Chained imports
+
+DVC supports importing data importing data that was imported into the source
+repo, as long as all repos in the import chain are accessible from the final
+destination repo. Each source repo's default DVC remote must also be accessible
+from the final destination repo.
+
+Consider an example where we start with two DVC repos, A and B, with the
+following contents:
+
+```
+/repo/a
+├── foo
+└── foo.dvc
+
+/repo/b
+├── bar
+└── bar.dvc
+```
+
+Now, we create a third DVC repo, C, with the following contents:
+
+```
+/repo/c
+└── dir
+    ├── bar
+    ├── bar.dvc
+    ├── foo
+    ├── foo.dvc
+    ├── subdir
+    │   └── baz
+    └── subdir.dvc
+```
+
+Repo C, contains a data directory tracked with `dvc add` (`dir/subdir`), as well
+as two separate imports (`dir/foo` and `dir/bar`). The imports were created via:
+
+```dvc
+# In repo C
+$ dvc import /repo/a foo -o dir/foo
+$ dvc import /repo/b bar -o dir/bar
+```
+
+If we examine the contents of `dir/foo.dvc`, we can see that it is imported from
+repo A:
+
+```yaml
+# dir/foo.dvc
+md5: d652071a8f0fd9f5c74d9348a468dec5
+frozen: true
+deps:
+- path: foo
+  repo:
+    url: /repo/a
+    rev_lock: 32ab3ddc8a0b5cbf7ed8cb252f93915a34b130eb
+outs:
+- md5: acbd18db4cc2f85cedef654fccc4a4d8
+  size: 3
+  path: foo
+outs:
+- md5: 630bd47b538d2a513c7d267d07e0bc44.dir
+  size: 3
+  nfiles: 1
+  path: subdir
+```
+
+Now, lets say that we want to create a fourth DVC repo, D, and run the following
+command:
+
+```dvc
+# In repo D
+$ dvc import /repo/c dir
+```
+
+This will result in the following directory structure, which contains two
+chained imports:
+
+- `foo` is imported from A into C into D
+- `bar` is imported from B into C into D
+
+```
+/repo/d
+├── dir
+│   ├── bar
+│   ├── foo
+│   └── subdir
+│       └── baz
+└── dir.dvc
+```
+
+If we examine `dir.dvc`, we can see that it only references repo C.
+
+```yaml
+# dir.dvc
+md5: bfdac3f7c77bdd89dcd1f6d22f5e39c5
+frozen: true
+deps:
+  - path: dir
+    repo:
+      url: /repo/c
+      rev_lock: 15136ed84b59468b68fd66b8141b41c5be682ced
+outs:
+  - md5: e784c380dd9aa9cb13fbe22e62d7b2de.dir
+    size: 27
+    nfiles: 4
+    path: dir
+```
+
+Each time that we run `dvc import dir` (from C), `dvc update dir` or
+`dvc pull dir`, DVC will first look up the contents of `dir` in repo C. At that
+point in time, DVC will see that `foo` and `bar` are chained imports from repos
+A and B, and then continue resolving the chain as needed (by looking up `foo`
+from A, and `bar` from B).
+
```

Comment:
I think the example can be simplified by having `A/foo.dvc` -> `B/foo.dvc` -> `C/foo.dvc`. No need for more than one file to show the chained import.

Reply:
Simplifying...